ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -174,6 +177,26 @@ class MySqlNamedBlobDb implements NamedBlobDb {
   private static final String TTL_UPDATE_QUERY =
       String.format("UPDATE %s SET %s, %s = NULL WHERE %s", NAMED_BLOBS_V2, STATE_MATCH, DELETED_TS, PK_MATCH_VERSION);
 
+
+  private static final String NEW_GET_STALE_QUERY = String.format(
+      "SELECT %s, %s, %s, %s, %s, %s, %s, %s " +
+          "FROM %s " +
+          "WHERE container_id = ? AND account_id = ? " +
+          "ORDER BY %s ASC, %s DESC " +  // <-- Changed this line
+          "LIMIT ? OFFSET ?;",
+      ACCOUNT_ID,
+      CONTAINER_ID,
+      BLOB_NAME,
+      BLOB_ID,
+      VERSION,
+      BLOB_STATE,
+      MODIFIED_TS,
+      DELETED_TS,
+      NAMED_BLOBS_V2,
+      BLOB_NAME,    // First order by blob_name alphabetically (ASC)
+      MODIFIED_TS   // Then order by modified_ts descending (DESC)
```

Comment:
We sort by modified_ts instead of version, it will lead to some equivalent value so we don't know which one is the old one?